CMakeLists.txt

```diff
@@ -7,8 +7,8 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 add_definitions(-Wall -Werror -flto)
 set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -flto")
-set(CMAKE_AR "llvm-ar")
-set(CMAKE_RANLIB "llvm-ranlib")
+set(CMAKE_AR "llvm-ar-5.0")
```

Comment:
I think it's better to set this to the system clang compiler version, otherwise  we could have problems on other operating systems. So maybe revert this? You can override the variables manually in your working copy if you need it on your machine in order to build.

Reply:
I disagree. Because we are specifying LLVM 5 with `find_package(LLVM 5 REQUIRED CONFIG)` we should also be using the llvm 5 version of all other llvm tools to avoid any compatibility issues.

Reply:
I actually hadn't meant to check that in, but now that Brandon mentions it, I think it's worth keeping. This keeps us safer.